pkg/backtest/matching.go

```diff
@@ -410,6 +410,15 @@ func (m *SimplePriceMatching) SellToPrice(price fixedpoint.Value) (closedOrders
 func (m *SimplePriceMatching) processKLine(kline types.KLine) {
 	m.CurrentTime = kline.EndTime.Time()
 	m.LastKLine = kline
+	if m.LastPrice.IsZero() {
+		m.LastPrice = kline.Open
+	} else {
+		if m.LastPrice.Compare(kline.Open) > 0 {
+			m.BuyToPrice(kline.Open)
```

Comment:
should be "sellTo" here? since last price > next open, the price is moving down?